src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -67,14 +67,17 @@
  (fn [{:keys [db]} [_ {:keys             [url-prefix
                                           enable-stmt-html
                                           enable-admin-status
-                                          no-val?]
+                                          no-val?
+                                          enable-admin-delete-actor]
                        ?oidc             :oidc
                        ?oidc-local-admin :oidc-enable-local-admin}]]
    (merge {:db (assoc db
                       ::db/xapi-prefix url-prefix
                       ::db/enable-statement-html enable-stmt-html
                       ::db/oidc-enable-local-admin (or ?oidc-local-admin false)
-                      ::db/enable-admin-status enable-admin-status)}
+                      ::db/enable-admin-status enable-admin-status
+                      ::db/enable-admin-delete-actor enable-admin-delete-actor
```

Comment:
I am not seeing a db.cljs spec entry for this field. I am surprised its not erroring from spec issue.

Reply:
looks like db spec is determined in terms of what it MUST have, not what it MUST NOT or MUST ONLY have.  I take it we want it required?

Reply:
I added it in